tethys_sdk/static/tethys_sdk/css/wide_nav.css

```diff
@@ -0,0 +1,28 @@
+/* Wider Nav */
```

Comment:
Yes, but it wouldn't be straightforward. At the moment this just adds the `wide_nav.css` stylesheet to the page when it is enabled. 

One way to make that configurable would be to use the new [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) functionality. We could replace the nav width with a CSS variable with a default value and then it could be easily updated in one place. We then need to decide how to expose the capability to users (app_class property, Layout view property, well known context variable?). 

I'd suggest we make an issue for this as a feature request that is not tied to this PR.